src/ws/cockpit-wsinstance-https@.socket.in

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Socket for Cockpit Web Service https instance %I
+PartOf=cockpit.service
```

Comment:
`PartOf` doesn't seem strong enough; the sockets stay around when cockpit-tls times out.  Replacing this with `BindsTo` seems to work.

Reply:
BindsTo has the effect that e. g. starting `cockpit-wsinstance-http.socket` manually will also activate cockpit.service. This feels a bit weird, but (1) it's not really harmful, and (2) who does that. I first tried with `StopWhenUnneeded=`,  but that's too strong. So indeed I agree that BindsTo= is right. This is also an issue on current master, although of course it's strictly bounded there. I fixed this with a test in #13014.

Reply:
I updated testUnitLifecycle to spot the https@ socket leak as well, fixed that, and pushed on top of #13011 and #13014.